packages-common/pluggable-widgets-tools/utils/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Let's move this one to the top of the package

Reply:
Moved it to the top.